openfl-tutorials/experimental/requirements_workflow_interface.txt

```diff
@@ -1,3 +1,4 @@
-dill==0.3.6
-metaflow==2.7.15
-ray==2.2.0
+dill==0.3.6
+metaflow==2.7.15
+ray==2.2.0
+numpy==1.21.6
```

Comment:
Is it necessary to pin the numpy version?

Reply:
We were having dependencies issues with the latest numpy version while combining the previously existing examples, pinning it allowed our 401 tutorial to run successfully